packages/common/src/api/user.ts

```diff
@@ -47,6 +54,40 @@ const parseTransaction = ({
 const userApi = createApi({
   reducerPath: 'userApi',
   endpoints: {
+    getUserAccount: {
+      fetch: async ({ wallet }: { wallet: string }, { audiusSdk }) => {
+        try {
+          const sdk = await audiusSdk()
+          const { data } = await sdk.full.users.getUserAccount({ wallet })
+          if (!data) {
+            console.warn('Missing user from account response')
+            return null
+          }
+
+          const account = accountFromSDK(data)
+          // If we got a valid account, populate user bank since that's
+          // expected to exist on "account" users
+          if (account) {
+            const userBank =
```

Comment:
just confirming - this requires no RPC call right (ie has no change of failing)

also - i would think that the backend could actually return this data on all users (but perhaps left to a follow up task)